kwanghwi/myapp/app.js

```diff
@@ -0,0 +1,52 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+const dotenv = require("dotenv");
+dotenv.config();
+
+const { DataSource } = require("typeorm");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource
+  .initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  })
+  .catch(() => {
+    console.log("Promise Rejected!");
+  });
+
+const app = express();
+const PORT = process.env.PORT;
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+// health check
+// http -v GET 127.0.0.1:3000/ping
+app.get("/ping", (req, res) => {
+  res.status(200).json({ message: "pong" });
```

Comment:
```suggestion
  return res.status(200).json({ message: "pong" });
```
화살표 함수로 작성된 실행문에서 더이상 실행할 로직이 없다면 return을 작성해주세요